README.md

```diff
@@ -403,10 +411,12 @@ Sorted by [year](#year) or [topics](#topics)
      `FSPN`
    - [[Poon2011](#poon2011)]
    [**Sum-Product Networks: A New Deep Architecture**](http://spn.cs.washington.edu/papers/spn.pdf) `SPN`
-	 
```

Comment:
I tried keeping the diff as small as possible. The tab character here seemed to be causing some issues in my editor, so I removed the line.